lib/internal/process/pre_execution.js

```diff
@@ -206,8 +205,7 @@ function patchProcessObject(expandArgv1) {
   let mainEntry;
   // If requested, update process.argv[1] to replace whatever the user provided with the resolved absolute file path of
   // the entry point.
-  if (expandArgv1 && process.argv[1] &&
-      !StringPrototypeStartsWith(process.argv[1], '-')) {
+  if (expandArgv1 && process.argv[1] && process.argv[1][0] !== '-') {
```

Comment:
```suggestion
  if (expandArgv1 && process.argv?.[1][0] !== '-') {
```

Reply:
Same, If the string is empty, we will hit undefined and a property chain lookup

Do you think it doesn't matter? Or can the strings not be empty here?

Reply:
That's not the same:
* The `?.` is misplaced. It should be `process.argv[1]?.[0] !== '-'`.
* If `process.argv[1] === undefined`, then `process.argv[1]?.[0] !== '-'` is `true` whereas the original version was `false`.

I suggest we keep the original change.

Reply:
It also covers the empty string, in case it's passed, to not hit the proto chain